macos/FluentUIUnitTest/AvatarViewTests.swift

```diff
@@ -115,13 +115,12 @@ class AvatarViewTests: XCTestCase {
 		XCTAssertNil(noNameNoEmailAvatar.toolTip)
 	}
 
-	func testBackgroundColors () {
+	func testAvatarColors () {
 		// Cherry pick a few known values and test them
-		XCTAssertEqual(AvatarView.backgroundColor(for: 0), Colors.Palette.cyanBlue10.color)
-		XCTAssertEqual(AvatarView.backgroundColor(for: 2), Colors.Palette.magenta20.color)
-		XCTAssertEqual(AvatarView.backgroundColor(for: 3), Colors.Palette.green10.color)
-		XCTAssertEqual(AvatarView.backgroundColor(for: 6), Colors.Palette.orange20.color)
-		XCTAssertEqual(AvatarView.backgroundColor(for: 20), Colors.Palette.orange30.color)
+		XCTAssertEqual(AvatarView.getColor(for: 0),
```

Comment:
The test I suggested is to iterate over all color sets and compare like the iOS one does it:

for each colorset:
assert -> Light mode foreground = Dark mode background
assert -> Dark mode foreground = Light mode background